tests/attacks/poison/test_feature_collision.py

```diff
@@ -66,29 +48,17 @@ def poison_dataset(classifier, x_clean, y_clean):
 
         return x_poison, y_poison
 
-    # def test_tensorflow(self):
-    #     """
-    #     First test with the TensorFlowClassifier.
-    #     :return:
-    #     """
-    #
-    #     tfc, sess = get_image_classifier_tf()
-    #     x_adv, y_adv = self.poison_dataset(tfc, self.x_train_mnist, self.y_train_mnist)
-    #     tfc.fit(x_adv, y_adv, nb_epochs=NB_EPOCHS, batch_size=32)
-    #
-    #     if sess is not None:
-    #         sess.close()
+    return _poison_dataset
 
-    def test_keras(self):
-        """
-        Test working keras implementation.
-        :return:
-        """
 
-        krc = get_image_classifier_kr()
-        x_adv, y_adv = self.poison_dataset(krc, self.x_train_mnist, self.y_train_mnist)
+@pytest.mark.skipMlFramework("non_dl_frameworks", "mxnet", "pytorch", "tensorflow")
+def test_poison(art_warning, image_dl_estimator, poison_dataset):
```

Comment:
Can we add a test to check if the call to `fit` is making any changes?

Reply:
Should this test be in the appropriate test for the classifier? I was thinking here that we're testing that the poison is the correct shape to be fit on. Did you have something else in mind?

Reply:
@ebubae I think my main concern was that this test is not checking if the attack is modifying anything at all, or?